core/templates/dev/head/css/oppia.css

```diff
@@ -686,11 +686,11 @@ textarea {
   }
   
   .oppia-footer-container {
-  max-width: 800px;
-  margin-left: auto;
-  margin-right: auto;
-  padding: 30px 80px 30px 30px;
-}
+    max-width: 800px;
```

Comment:
The following does not make a difference when they are toggled on/off in the inspector so it would be better to delete them.

```
max-width: 800px;
margin-left: auto;
margin-right: auto;
```
It is merely overwriting what's already declared in the `.oppia-footer-container` outside the media query. It will inherit those styles so its not a problem to just delete the duplicated code.

The padding can be simplified to just `30px` on all sides; it doesn't make any sense to have `80px` on the left.

Also, it would help if you could make the commit messages more specific to the changes made.

Thanks! :+1: